db/schema.rb

```diff
@@ -20,6 +20,22 @@
     t.text     "name"
     t.datetime "created_at"
     t.datetime "updated_at"
+    t.integer  "black_player_id"
+    t.integer  "white_player_id"
+    t.integer  "winner_id"
+    t.integer  "current_player_turn"
+  end
+
+  create_table "pieces", force: true do |t|
+    t.string   "type"
+    t.integer  "current_row_index"
+    t.integer  "current_column_index"
+    t.string   "color"
+    t.boolean  "captured?"
```

Comment:
i'd recommend changing this to just `captured`

Then since it is a boolean, in Rails you'll have `captured?` available to you.